### ISSUE TITLE
No module named 'simtk.openmm.app.topology'

### ISSUE THREAD

**Issue by dotsdl** [choderalab/fah-xchem#160]
Observing the following in logs in `dev` environment:

```
Exception occurred extracting snapshot from /home/server/server2/projects/13456 data /home/server/server2/data/SVR916310614/PROJ13456 run 943 clone 29 gen 1
No module named 'simtk.openmm.app.topology'
```
I believe this is related to the use of `simtk` imports in `mdtraj`; created mdtraj/mdtraj#1695. May need to resolve to eliminate these, or find a way to avoid the issue.

**Comment by jchodera** [comment]
With OpenMM 7.6, these imports should move to `openmm.app.topology`.  We can update the imports in `fah-xchem`, but in codes like MDTraj that need to maintain backward compatibility, we should try both in a `try..except`. Any chance you could help with this?

**Comment by dotsdl** [comment]
Yes, I can take this on in `mdtraj` itself. It needs to happen anyway!

**Comment by dotsdl** [comment]
This may actually be due to `perses`; there are imports exactly like the one in the logs above in current `master` there. We have an existing issue in choderalab/perses#837.
